### DISCUSSION TITLE
Contributor Call: 17 September 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2720] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 17 September 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/sep-17-2022/4pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda
TBD

> Notes will be added after the call.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark
- @KaerMorhan 
- @bobgeorgethe3rd 
- @lexander2002 
- @BenJamesBen 
- @bbdapalkow
- @SeaZeeZee 
- @starfetch 
- illagerBill9

✨ **Lexander**
Lexander will be presenting FreeSewing at a workshop at the [Rotterdam Expo](https://serendiep.nl/nl/expo_rotterdam2022/lineup#id62fddf93bfb45).

✨ **Karen**
Showing off a gorgeous Aaron dress. Still pondering about creating a guide for it.

✨ **Maya&Morgan**
Ukrainian translation hit 7% :^)
_Question for Joost: are there things from v2 that won't migrate to v3 and should be deprioritized for translation?
Answer: almost certainly. But we won't really know what will be dropped until (the frontend for) v3 is ready. So unless you one can predict the future 🤷_

✨ **Benjamin**
Benjamin is considering working on a new FreeSewing pattern: pirate shirt! @SeaZeeZee had a similar idea so collaboration is possible.
Another small project: a 17-18 century shirt-waist/bodice. ([Dressmaking: a manual for schools and colleges](https://digital.library.cornell.edu/catalog/hearth4216153), p.174)

✨ **Karen**
Karen is pondering on the idea of creating a robe pattern (that one from Our Flag Means Death).
Are there any ideas for the newsletter? Please let drop it in the [#newsletter-blogs](https://discord.com/channels/698854858052075530/823470759019282432) text channel!
(BTW, the interview with Enoch is coming soon).
_Some of the ideas: contacting Wouter about Hi and Octoplushy; Lexander writing a short bit about the Rotterdam Expo; linking previous Hallowing ideas with illustrations._

**Reply by starfetch** [reply]
Just a quick note that you can also tag me here on GitHub :) -- starf aka starfetch

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
